app/components/AlbumArt/index.js

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components';
```

Comment:
this shouldn't be called AlbumArt anymore, lets give it a generic name

Reply:
will add a generic name like LazyImage